packages/sku/telemetry/provider.js

```diff
@@ -1,3 +1,6 @@
+const { packageManager, isYarn } = require('../lib/packageManager');
+const { getCommand } = require('@antfu/ni');
```

Comment:
I was thinking it would be ideal to absorb `@antfu/ni` into `lib/packageManager` so there's only one way to interface with the package manager. But we can iterate on that later.

Reply:
Good idea. I've got a follow up PR that will use `getCommand` a bit more, I can refactor it there.